### PR TITLE
Restored the search bar with some stylistic improvements.

### PR DESCRIPTION
**Pull request by AngeloD2022**
CleanShot 2023-07-18 at 13 06 04@2x

### CODE REVIEW

File: inspector/templates/base.html [hunk 1]
```diff
@@ -1,59 +1,69 @@
 <html>
-  <head>
+<head>
```
Comment (di) [inline inspector/templates/base.html:2]: @AngeloD2022 Seems like there's some unnecessary formatting/indenting changes in this PR like this, would you mind reverting those?

**Comment by di** [comment]
Hey @AngeloD2022, still seeing lots of indentation-only changes here, I think it's likely because of a difference between four-space and two-space indents by default.

**Comment by AngeloD2022** [comment]
Sorry about this. I shouldn't have formatted it.

**Comment by AngeloD2022** [comment]
Thank you
